src/pull-request/reviewers-list.html

```diff
@@ -0,0 +1,58 @@
+<dom-module id="reviewers-list">
+  <template>
+    <style>
+      :host {
+        display:flex;
+        flex-wrap: wrap;
+      }
+      .lgtm-true.false {
+        display: none;
+      }
+      iron-icon {
+        -webkit-box-shadow: 0 0 0 3px rgba(0,0,0,0.2);
+        box-shadow: 0 0 0 3px rgba(0,0,0,0.2);
+        padding: 2px;
+        width:16px;
+        height:16px;
+        background-color: var(--primary-background-color);
+        border-radius: 9px;
+        margin-left: -9px;
+        margin-top: -9px;
+      }
+      .lgtm-true {
+        fill: var(--primary-action-green);
+      }
+      .reviewer {
+        -webkit-box-shadow: 0 0 4px 0 rgba(0,0,0,0.2);
+        box-shadow: 0 0 4px 0 rgba(0,0,0,0.2);
+        border-radius: 9px;
+        width: 52px;
+        height: 52px;
+        background-repeat: no-repeat;
+        background-size: 100%;
+        margin: 8px 8px;
+      }
+    </style>
+
+    <template is="dom-repeat" items="[[pullRequest.reviewers]]" as="reviewer">
+      <div class="reviewer" style$="order: [[_lgtmToOrder(reviewer.lgtm)]]; background-image: url([[reviewer.profile]]); ">
```

Comment:
This is because databinding does not work inside a stylesheet. As for using `<img>`, in my experience `img` elements tend to misbehave very quickly when you apply css to them.